check/check.go

```diff
@@ -60,21 +63,29 @@ func handleError(err error, context string) (errmsg string) {
 // Check contains information about a recommendation in the
 // CIS Kubernetes 1.6+ document.
 type Check struct {
-	ID             string      `yaml:"id" json:"test_number"`
-	Text           string      `json:"test_desc"`
-	Audit          string      `json:"audit"`
-	Type           string      `json:"type"`
-	Commands       []*exec.Cmd `json:"omit"`
-	Tests          *tests      `json:"omit"`
-	Set            bool        `json:"omit"`
-	Remediation    string      `json:"remediation"`
-	TestInfo       []string    `json:"test_info"`
+	ID             string        `yaml:"id" json:"test_number"`
+	Text           string        `json:"test_desc"`
+	Audit          string        `json:"audit"`
+	AuditCommands  AuditCommands `yaml:"audit_commands"`
+	Type           string        `json:"type"`
+	Commands       []*exec.Cmd   `json:"omit"`
+	Tests          *tests        `json:"omit"`
+	Set            bool          `json:"omit"`
+	Remediation    string        `json:"remediation"`
+	TestInfo       []string      `json:"test_info"`
 	State          `json:"status"`
 	ActualValue    string `json:"actual_value"`
 	Scored         bool   `json:"scored"`
 	ExpectedResult string `json:"expected_result"`
 }
 
+type AuditCommands struct {
+	FromConfig string      `yaml:"from_config"`
+	FromParams string      `yaml:"from_params"`
+	ConfigCmds []*exec.Cmd `json:"omit"`
+	ParamsCmds []*exec.Cmd `json:"omit"`
+}
```

Comment:
I think it would be neater to just add a new pair of fields into the original `Check` struct so that it's more like 

```
...
Audit       string `json:"audit"`
AuditConfig string `json:"audit_config"`
...
Commands []*exec.Cmd `json:"omit"`
ConfigCmds []*exec.Cmd `json:"omit"`
...
```

That way the back-compatibility comes essentially for free - if there is no `audit_config` field in the test definition you can simply use exactly the code as-is